core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/cluster/CloudStorageConverter.java

```diff
@@ -178,7 +178,7 @@ public AwsEfsParameters fileSystemToEfsParameters(FileSystem fileSystem) {
 
         efsParameters.setName(fileSystem.getName());
 
-        Map<String, String> configurations = null;
+        Map<String, Object> configurations = null;
         try {
             configurations = JsonUtil.readValue(fileSystem.getConfigurations().getValue(), Map.class);
         } catch (IOException ex) {
```

Comment:
In other places when reading json failed, it throws exception. Since EFS is not essential for stack, I decide to log error message, and return null. So do not create EFS instance. When we move some services to EFS, I will change it to fail the stack creation.